src/integ_test/java/games/strategy/engine/lobby/server/login/LobbyLoginValidatorTest.java

```diff
@@ -84,33 +85,12 @@ public void testAnonymousLogin() {
   @Test
   public void testAnonymousLoginBadName() {
     final String name = "bitCh" + Util.createUniqueTimeStamp();
-    new BadWordController().addBadWord("bitCh");
```

Comment:
This is an interesting problem, it works okay for Derby, or Travis, but not in a dev env.

In a dev env we have the same tables between runs, so you get an insertion error. In derby or travis, this would be inserted only once for the one test run and everything is then okay. Hence why it was removed, yet is still needed for travis buils.